session/pingpong/factory.go

```diff
@@ -24,27 +24,54 @@ import (
 	"github.com/mysteriumnetwork/node/communication"
 	"github.com/mysteriumnetwork/node/core/connection"
 	"github.com/mysteriumnetwork/node/core/node"
-	"github.com/mysteriumnetwork/node/eventbus"
 	"github.com/mysteriumnetwork/node/identity"
 	"github.com/mysteriumnetwork/node/market"
 	"github.com/mysteriumnetwork/node/money"
-	"github.com/mysteriumnetwork/node/services/openvpn/discovery/dto"
 	"github.com/mysteriumnetwork/node/session"
-	"github.com/mysteriumnetwork/node/session/balance"
-	payment_factory "github.com/mysteriumnetwork/node/session/payment/factory"
-	"github.com/mysteriumnetwork/node/session/promise"
 	"github.com/mysteriumnetwork/payments/crypto"
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
 )
 
+// PromiseWaitTimeout is the time that the provider waits for the promise to arrive
+const PromiseWaitTimeout = time.Second * 50
+
+// InvoiceSendPeriod is how often the provider will send invoice messages to the consumer
+const InvoiceSendPeriod = time.Second * 60
+
 // DefaultAccountantFailureCount defines how many times we're allowed to fail to reach accountant in a row before announcing the failure.
 const DefaultAccountantFailureCount uint64 = 10
 
-// DefaultPaymentInfo represents the default payment info for the alpha release
-var DefaultPaymentInfo = dto.PaymentRate{
+// DefaultPaymentMethod represents the the default payment method of time + bytes.
+// The rate is frozen at 0.07MYSTT per GB of data transfered and 0.0005MYSTT/minute.
+var DefaultPaymentMethod = PaymentMethod{
 	Price:    money.NewMoney(50000, money.CurrencyMyst),
-	Duration: 1 * time.Minute,
+	Duration: time.Minute,
+	Type:     "BYTES_TRANSFERED_WITH_TIME",
+	Bytes:    7142857,
```

Comment:
Not clear what kind of number is here. And I don't see expression of 0.07MYSTT per GB

Reply:
added explanation